docs/source/markdown/podman-image-mount.1.md

```diff
@@ -1,24 +1,29 @@
 % podman-image-mount(1)
 
 ## NAME
+
 podman\-image\-mount - Mount an image's root filesystem
 
 ## SYNOPSIS
-**podman image mount** [*options*] [*image* ...]
+
+**podman image mount** [*options*] name
+**podman mount** [*options*] name
```

Comment:
This is not correct `podman mount` is used for containers. This page is about `podman image mount`.

Reply:
The synopsis was correct. The input of images after the command or the options is only required when the `--all` option isn't used. And I believe that image is the better term here.

Reply:
> The synopsis was correct. The input of images after the command or the options is only required when the --all option isn't used. And I believe that image is the better term here.

I would revert this comment.

I added the following definition for cases like these into the MANPAGE_SYNTAX:

> Many manpages include the OPTIONS **--all**, **-a** and/or **--latest**, **-l**. In this case there is no `container name` or `ID` needed after the initial command. Because most of the other OPTIONS still need the `container name` or ` ID`, it is defined that the *container* argument in the command should **not** be put in brackets. It should also be noted in the *IMPORTANT* section in the DESCRIPTION of the OPTION.

I believe this is a good way to work with  **--all**, **-a** and **--latest**, **-l** without compromising the command structure and keeping the general idea behind it intact.